src/uiTest/resources/project.rhino.js

```diff
@@ -237,4 +245,12 @@ function Project(underlyingProject) {
     if (hash === null) return null;
     return hash.asString();
   };
+
+  this.getSyncToParentStatus = function (parent, child, status) {
+    const snapshot = getGraphTable().getGitMacheteRepositorySnapshot();
+
+    return snapshot.getManagedBranchByName(parent)
+      .flatMap(p => p.getChildren().find(c => c.getName().equals(child)))
+      .map(b => b.getSyncToParentStatus().name()).getOrElse('');
```

Comment:
Actually... won't it be easier to just `snapshot.getManagedBranchByName(child)` and `getSyncToParentStatus` on this child? 🤔 with `parent` **not** even passed to the method

Reply:
good point